frameworks/digital-outcomes-and-specialists-2/manifests/briefs_search_filters.yml

```diff
@@ -1,10 +1,12 @@
+-
+  name: Status
+  questions:
+    - status
 -
   name: Location
   slug: location
   questions:
-    - locationOutcomes
-    - locationParticipants
-    - locationSpecialists
+    - locationAllLots
```

Comment:
If this became a regular thing, it might be nicer to have a way of doing this automatically, e.g. (and you'll notice I haven't thought this through at all) something like `ignore-depends-on-restriction: ["lot"]`.

But we certainly don't need to bother if the only example is now this one field.

Reply:
Yeah, I agree. Have had a look through though and this is the only example of this I can find.

Reply:
Is that not just `location` if the question is identical for all lots?

Reply:
The only reason for splitting questions by lot is if they have different wording.

Reply:
Hey Kev, the locations for specialists and outcomes are slightly different to user research participants. They have 'offsite', and user research participants has 'international'. So we need the lot specific questions for when creating a brief.

It was decided that rather than having lot specific location filters, which would cause the available locations to filter by to change when moving between lots, that there should be one filter which covers all of the available options. Even if that means that you always get 0 results for user research participants if applying the offsite filter, for example.

This is the question that powers the filter, with all values and not depending on lot. Do you think there might be a better way of doing it?

You might be right about calling it `location`, although i was following the pattern of <question><lot>.